### ISSUE TITLE
link to BIDS 2.0

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/bids-website#396]

**Comment by Remi-Gau** [comment]
@yarikoptic 
are there plans to create a BIDS 2.0 working group?

**Comment by yarikoptic** [comment]
Yes. We have a "github team" already https://github.com/orgs/bids-standard/teams/bids-2-0 and I should get back to review what other formalities should be done to announce that it exists...

**Comment by Remi-Gau** [comment]
Just so we do not forget: info about it can be added there

https://github.com/bids-standard/bids-website/blob/887cd74b4732776c66148cedc29881d9c9df4d65/data/working_groups.yml#L21-L39
